PlayerPanel.js

```diff
@@ -34,87 +34,121 @@ function update_pclist() {
+	const addPartyButton = $("<button id='add-party'>ADD PARTY</button>");
+	addPartyButton.on('click', () => {
+		window.pcs.forEach(function (p, i) {
+			token_button({ target: $(`[data-name='${p.name}']`) }, i, window.pcs.length);
+		});
+	});
+	addPartyButtonContainer.append(addPartyButton);
+	pcs_list.append(addPartyButtonContainer);
 
 	var NEXT_COLOR = 0;
 	window.pcs.forEach(function(item, index) {
 
 		color = "#" + TOKEN_COLORS[NEXT_COLOR++];
 
 		pc = item;
-		row = $("<div/>");
-		row.css("padding", "20px");
-
-		player_name = $("<div/>");
-		player_name.text(pc.name).css("background", color).css("font-weight", "bold");
-		player_name.attr('data-nextcolor', NEXT_COLOR);
-		row.append(player_name);
-
-		row_inside = $("<div/>");
-		row_inside.css('border', '1px solid black');
-		row_inside.css("width", "100%");
-		row_inside.css("float", "left");
-		row.append(row_inside);
-
-		row_inside.append($("<div style='float:left;'><img width=60 height=60 src='" + pc.image + "'></div>"));
-
-		stats = $("<div/>");
-		stats.css("float", "left");
-
-		stats.append("<div><div><b>HP:</b></div><div class='hp'>-</div></div>");
-		stats.append("<div><div><b>AC:</b></div><div class='ac'>-</div></div>");
-		stats.append("<div><div><b>Passive Perception:</b></div><div class='pp'>-</div></div>");
-
-		row_inside.append(stats);
-
-		conditions = $("<div style='float:left;'/>");
-		conditions.append("<div><b>Conditions</b></div>");
-		conditions.append("<div class='conditions'></div>");
-
-		row_inside.append(conditions);
 
+		let playerData;
 		if (pc.sheet in window.PLAYER_STATS) {
-			data = window.PLAYER_STATS[pc.sheet];
-
-			stats.find(".hp").text(data.hp + "/" + data.max_hp);
-			stats.find(".ac").text(data.ac);
-			stats.find(".pp").text(data.pp);
-
-			for (var i = 0; i < data.conditions.length; i++)
-				conditions.find(".conditions").append($("<div/>").text(data.conditions[i]));
+			playerData = window.PLAYER_STATS[pc.sheet];
 		}
+		
+		newPlayerTemplate = `
+			<div class="player-card">
+				<div class="player-card-header">
+					<div class="player-name">${pc.name}</div>
+					<div class="player-actions">
+						<button 
+							data-set-token-id='${pc.sheet}' data-img='${pc.image}' data-name="${pc.name}"
+							data-hp="${playerData ? playerData.hp : ''}"
+							data-ac="${playerData ? playerData.ac : ''}"
+							data-maxhp="${playerData ? playerData.max_hp : ''}"
+							data-color="${color}" class="add-token-btn">
+							TOKEN
+						</button>
+						<button data-target='${pc.sheet}' class="open-sheet-btn">SHEET</button>
+					</div>
+				</div>
+				<div class="player-card-content">
+					<div class="player-token" style="box-shadow: ${
+						playerData ? `${token_health_aura(
+							Math.round((playerData.hp / playerData.max_hp) * 100)
+						)} 0px 0px 11px 3px` : 'none'
+					};">
+						<img width="70" height="70" src="${pc.image}" style="border: 2px solid ${color}" />
+						${
+							playerData ? `

```

Comment:
Another way is to short circut this with just `&&` so you don't have an empty ternary. This is a style thing, so don't worry about it. Just bringing it up as a common practice thing